autopush/web/webpush.py

```diff
@@ -43,12 +44,13 @@ def post(self, api_ver="v1", token=None):
             headers=self.valid_input["headers"],
             ttl=self.valid_input["headers"]["ttl"]
         )
-
+        self._client_info["uaid"] = hasher(user_data.get("uaid"))
```

Comment:
nope, removed the uaid setter for web/*. Removed the chid setter too. We're not using it and the only thing it was doing that was constructive was rendering this hash. So moved it to the initialize instead. We normalize the chid in db.py before we write it.